### PR TITLE
SendData Server API

### PR DESCRIPTION
**Pull request by ChesterMing**

### CODE REVIEW

File: livekit_room.proto [hunk 1]
```diff
@@ -125,3 +128,14 @@ message UpdateSubscriptionsRequest {
 message UpdateSubscriptionsResponse {
   // empty for now
 }
+
+message SendDataRequest {
+  string room = 1;
+  repeated string destination_sids = 2;
```
Comment (davidzhao) [inline livekit_room.proto:134]: nit: since destination_sids is optional, do you mind placing this last?

**Review by davidzhao — COMMENTED** [review]
looks good, just a small nit

File: livekit_room.proto (hunk 1 above)
Comment (ChesterMing) [reply livekit_room.proto]: I made some changes, does it look better?

Comment (ChesterMing) [reply livekit_room.proto]: Is the name `destination_sids` ok? or do you have suggestions. 😁

**Review by davidzhao — APPROVED** [review]
looks good!

**Pull request merged**